src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsResponseDetails.java

```diff
@@ -35,49 +35,18 @@ public static List<FeedbackResponseAttributes.UpdateOptions> getUpdateOptionsFor
 
         FeedbackResponseDetails details;
         FeedbackRankRecipientsResponseDetails responseDetails;
```

Comment:
I believe that variables should be declared in the smallest scope as possible. `details` and `responseDetails` are only used in the for-loop at line 52. Would it be possible to shift the variable declarations there?